### DISCUSSION TITLE
How to pass frames to the stream lit from inference pipeline?

### DISCUSSION THREAD

**Discussion by MubashirWaheed** [roboflow/supervision#1396] [Q&A]
After running inference through inference pipeline I want to pass the frames to the streamlit so that it can be displayed on the web. 
how can this be done? Is there a working example of this or a blog post. I checked but didn't find it.

**Comment by SkalskiP** [comment]
@capjamesg is there a chance you've worked with the InferencePipeline + Streamlit combo?

**Reply by MubashirWaheed** [reply]
It doesn't have to be streamlit. The goal is to have the inference stream on a webpage. I tried sending through flask but I am getting 1-2 fps. Also I am not sending the stream over internet it is all on the local network and thinking of deploying on the edge device when I have it working

**Reply by SkalskiP** [reply]
@MubashirWaheed, so the aim is to somehow extract subsequent frames from `InferencePipeline`?

**Reply by MubashirWaheed** [reply]
@SkalskiP  Yes that's the first part

**Reply by SkalskiP** [reply]
@MubashirWaheed I reached out to the Inference team; I hope they will come up with some solution.

**Reply by MubashirWaheed** [reply]
@SkalskiP I posted on the roboflow forum and was able to have the stream working with the streamlit but due to some constraints from the streamlit side I wasn't able to get the desired fps(I was getting 10 and requirement was 18-20fps)  so I dropped it. 
You can read the thread here: https://discuss.roboflow.com/t/how-to-imbed-the-video-frames-from-the-inference-pipeline-in-the-web/6597/23
I am still searching for solution

**Reply by MubashirWaheed** [reply]
I am really surprised that their hasn't been a streamlined way of doing this. I searched but haven't found the solution
